src/cookieConsents/cookieContentSource.ts

```diff
@@ -0,0 +1,118 @@
+import { ContentSource } from 'hds-react';
+import { TFunction } from 'react-i18next';
+
+export const trackingCookieId = 'matomo';
+
+export function getCookieContent(
+  t: TFunction
+): Pick<
+  ContentSource,
+  'requiredCookies' | 'optionalCookies' | 'texts' | 'siteName'
+> {
+  const securityCookieDescription = t('cookies.securityCookieDescription');
+  const loadBalancerCookieName = t('cookies.loadBalancerCookieName');
+  const loadBalancerCookieDescription = t(
+    'cookies.loadBalancerCookieDescription'
+  );
+  const languageCookieDescription = t('cookies.languageCookieDescription');
+  const trackingCookieDescription = t('cookies.trackingCookieDescription');
+  const consentStorageDescription = t('cookies.consentStorageDescription');
+  const sessionExpiration = t('cookies.sessionExpiration');
+  const daysPlural = t('cookies.daysPlural');
+  const minutesPlural = t('cookies.minutesPlural');
+  const mainText = t('cookies.mainText');
+  const siteName = t('appName');
+
+  const tunnistusHostName = 'tunnistus.hel.fi';
+  const tunnistamoHostName = 'tunnistamo.hel.fi';
```

Comment:
I don't know if it makes a difference here, but Tunnistamo isn't (and probably never will be) reachable via tunnistamo.hel.fi URL.